imports/plugins/included/connectors-shopify/server/methods/api/products.js

```diff
@@ -0,0 +1,46 @@
+import Shopify from "shopify-api-node";
+
```

Comment:
Are we spacing here because we're grouping? That's hard to be consistent on.

Reply:
yeah, I was grouping into Node, then Meteor, then Reaction, then local imports, but agreed and not sure it's worth the effort. I'll fix.

Reply:
yes, I was grouping by Node > Meteor > Reaction > local, but agree that it's not necessarily consistent with the rest of the code base, I can remove the spacing.